templates/ecommerce-3/src/access/adminsOrPublished.ts

```diff
@@ -0,0 +1,15 @@
+import type { Access } from 'payload'
+
+import { checkRole } from '@/access/checkRole'
+
+export const adminsOrPublished: Access = ({ req: { user } }) => {
+  if (user) {
+    return checkRole(['admin'], user)
```

Comment:
I know this is a draft PR, but shouldn't this be like this:

```ts
if (user) {
  if (checkRole(['admin'], user)) { 
    return true
  }
}
```

If you are logged in, but not an admin, we still need to check if the page status is `published` or not.

The current version will return `false` for a logged in normal user even if the page status is marked as `published`

Reply:
Thanks, a lot in this template will be revised to match the website template's best practices as well